wildlifelicensing/apps/dashboard/views/officer.py

```diff
@@ -297,7 +297,7 @@ def _render_action_column(obj):
             return '<a href="{0}">Issue Licence</a>'.format(
                 reverse('wl_applications:issue_licence', args=[obj.pk]),
             )
-        elif obj.processing_status == 'issued' and obj.licence is not None and obj.licence.licence_document is not None:
```

Comment:
Yeah, cos the licence / licence document don't need to be there to look at the application in read-only.